### PR TITLE
Add gradle wrapper

### PR DESCRIPTION
**Pull request by paraita**
This is to add basic support for Gradle. The `clean`task shall remove the npm and bower dependencies as well as the resulting dist folder. `build` will build the (default) enterprise version of the dashboard.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+// Configure the maven repository deployment
+install {
+    repositories.mavenInstaller {
+        // Set the version
+        pom.version = proactiveDashboardVersion
+        // Set the group/namespace for the maven repository deployment.
+        pom.groupId = 'org.ow2.proactive'
+        // Give the artifact a 'base name' (The version is added to the 'base name')
+        pom.artifactId = 'proactive-dashboard'
```
Comment (jrochas) [inline build.gradle]: Are you sure that specifying `pom.artifactId` is enough to force the name in all building environments?
Some of our other projects use the global variable `archivesBaseName`, e.g. we would have here `archivesBaseName = 'proactive-dashboard'`

File: gradle.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+proactiveDashboardVersion=7.34.0-SNAPSHOT
```
Comment (jrochas) [inline gradle.properties]: Will the version be updated automatically at each release, like our other projects?

File: build.gradle (hunk 1 above)
Comment (paraita) [reply build.gradle]: I believe this artifactId is not really used (we don't publish web-based projects to maven). This gradle configuration is based on the cloud-automation project. I'll try to remove the parts that are not relevant later on !

File: gradle.properties (hunk 2 above)
Comment (paraita) [reply gradle.properties]: Yes gradle handles the version bump after the release is done in all the repositories. This new service should follow the same road.

**Pull request merged**